l10n/en/firefox/features/translate.ftl

```diff
@@ -9,18 +9,18 @@ features-translate-translate-a-webpage-with-firefox = Translate a webpage with {
 features-translate-translate-the-web = Translate the web
 
 # HTML page description
-features-translate-firefox-translations-is-an-added = Translations is an added built-in translation feature that allows you to easily browse the web in your preferred language. Learn more about how this feature in { -brand-name-firefox } works, and how { -brand-name-mozilla } helps keep what you translate private.
+features-translate-firefox-translations-is-an-added = { -brand-name-firefox } Translation is a built-in translation feature that allows you to easily browse the web in your preferred language. Learn more about how this feature in { -brand-name-firefox } works, and how { -brand-name-mozilla } helps keep what you translate private.
```

Comment:
I understood from [Peiying's comment](https://github.com/mozilla-l10n/www-l10n/pull/379#discussion_r1506397732) that `Translation` should be pluralized to `Translations`?